### PR TITLE
[vsphere-csi] allow to configure env vars and extra mounts for all controller containers at once

### PR DESCRIPTION
**Pull request by sathieu**
(on the road to #49, see #50 for the full implementation)

### CODE REVIEW

**Comment by sathieu** [comment]
@eminaktas @MaxRink Please review 🙏 .

File: charts/vsphere-csi/values.yaml [hunk 1]
```diff
@@ -208,6 +208,25 @@ controller:
     ## Enable debug mode
     ##
     debug: false
+
+  allcontainers:
```
Comment (MaxRink) [inline charts/vsphere-csi/values.yaml:212]: not to keen on that naming, i would usually prefer "global"
@mylesagray your opinion?

either way, i would like to stick to CamelCase here, so 'allContainers'

**Review by MaxRink — CHANGES_REQUESTED** [review]

File: charts/vsphere-csi/values.yaml (hunk 1 above)
Comment (mylesagray) [reply charts/vsphere-csi/values.yaml:212]: Agree with @MaxRink on the use of `allContainers` - `global` is more universal

Comment (sathieu) [reply charts/vsphere-csi/values.yaml]: @MaxRink @mylesagray Comment addressed. Please review again :pray: .

**Comment by sathieu** [comment]
@MaxRink @mylesagray I've addressed your comments, please review again :pray:.

**Review by MaxRink — APPROVED** [review]

**Pull request merged**